Tools/Scripts/build-webkit

```diff
@@ -226,11 +226,9 @@ if ((isAppleWebKit() || isWinCairo() || isPlayStation() || isFTW()) && !-d "WebK
 my @options = ();
 
 if (isAppleCocoaWebKit()) {
-    # Ignore any workspace set by set-webkit-configuration
     if (!$useWorkspace) {
+        # Ignore any workspace set by set-webkit-configuration
         overrideConfiguredXcodeWorkspace("");
-    } else {
-        overrideConfiguredXcodeWorkspace("WebKit.xcworkspace");
```

Comment:
This represents a slight behavior change: Instead of _always_ using WebKit.xcworkspace, `build-webkit` may now use other configured workspaces. For internal engineers, this means that `build-webkit` will share incremental build data with other workspaces.

Reply:
Not sure if this is a scenario that is or should be supported. This change is a no-op for open source builds AFAICT, so it's OK.

Reply:
I know we document `build-webkit` as the preferred workflow for open source builds exclusively, but I think that using the configured workspace will make it do the expected thing for engineers running it internally. It also aligns `build-webkit`'s behavior with the other `build-*` scripts here.

You are correct that it's a no-op for open source builds.